docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,284 @@
+* `CA_URL`: The URL of the CA, followed by the port `7054` (unless the default port has been changed).
+* `CA_ADMIN`: The path to the location of the certificates for the admin of the CA the identity is being registered with.
+* `ID_TYPE`: The type (or role) of the identity. There are four possible types: `peer`, `orderer`, `admin`, and `client` (used for applications). This type must be linked to the relevant [NodeOU](#specifiying-nodeous). If NodeOUs are not being used, you can ignore the type and `--id.type` flag.
+* `ID_ATTRIBUTE`: Any attributes specified for this identity. For more information about attributes, check out [Attribute based access control](https://hyperledger-fabric-ca.readthedocs.io/en/latest/users-guide.html#attribute-based-access-control). These attributes can also be added as a JSON array, therefore the `$ID_ATTRIBUTE` is not meant to represent a single attribute but any and all attributes, which should be placed in the register command after the `--id.attrs` flag.
+* `TLSCERT`: The relative path to your the TLS CA root signed certificate (generated when creating the TLS CA).
+* `CSR_HOSTNAME`: Any hostname you wish you use. If you do not specify a CSR hostname, the default hostname, as specified in your CA configuration, will be used.
+
+Here is a sample register command for an admin identity:
+
+```
+./fabric-ca-client register -d --id.name org1admin --id.secret org1adminpw -u https://example.com:7054 --mspdir ./org1-ca/msp --id.attrs admin --tls.certfiles ../tls/tls-ca-cert.pem --csr.hosts 'host1,*.example.com'
+```
+
+After the identity has been successfully registered, the CA admin would give the enroll ID (`org1admin`) and enroll secret (`org1adminpw`) to the user who will enroll as an admin.
+
+**If you are creating the certificates needed for a node, make sure to register and enroll with the TLS CA associated with the organization as well.**
+
+## Enroll an identity
+
+Once the enrollment CA has been set up and identities have been registered, the CA admin will need to contact the user who will be enrolling out of band to give them the enroll ID and secret they used when registering the identity. Then, using this ID and secret, the user can enroll the identity using their own copy of the Fabric CA client to contact the relevant CA (which will be either an organization CA, used to create admin and node identities, or a TLS CA, used to generate the TLS certificates that nodes need). Note that if TLS has been enabled, this user will need to acquire the TLS CA root signed cert to include when enrolling.
+
+While it's possible to enroll a node identity before enrolling an admin, it makes more sense to enroll an admin first and establish your organization's MSP before enrolling it's dependent node (whether it's a peer or an ordering node). You certainly need to enroll an admin identity and place its certificate in the local MSP of a node before starting node.
```

Comment:
`before enrolling it's dependent node` -> `before enrolling its dependent node`

Reply:
`local MSP of a node before starting node` -> `local MSP of a node before starting the node`